app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -93,10 +94,24 @@
     private int chartZoom = 3;
     private boolean hasZoomedChart = false;
 
-    private NumberFormat sgvFormatter;
-    private boolean mmolxl;
+    public static NumberFormat sgvFormatter;
```

Comment:
Your right yeah - the only static I needed was mmolxl and mmolxlDecimals and these are back to private and the conversion is now part of main.